VideoLocker/src/main/java/org/edx/mobile/view/adapters/BaseListAdapter.java

```diff
@@ -125,6 +125,15 @@ public void replace(T item, int position) {
         notifyDataSetChanged();
     }
 
+    public void moveToTop(int itemPosition) {
+        if (itemPosition < objects.size()) {
+            T item = objects.get(itemPosition);
+            objects.remove(itemPosition);
+            objects.add(0, item);
```

Comment:
I actually swapped the items initially, but thats wrong logically, cuz we don't want 2 items to exchange places.